src/sass/modules/_m-schemaform.scss

```diff
@@ -498,3 +498,41 @@ fieldset.schemaform-field-template {
 .confirmation-guidance-message {
   margin-top: 0;
 }
+
+ul.react-autosuggest__suggestions-list {
+  background: $color-white;
+  border: 1px solid $color-gray;
+  // Box shadow only sides and bottom of suggestion list.
+  box-shadow: 0px 3px 3px -3px $color-focus,
+              3px 0 3px -3px $color-focus,
+              -3px 0 3px -3px $color-focus,
+              0 7px 7px -7px $color-focus,
+              7px 0 7px -7px $color-focus,
+              -7px 0 7px -7px $color-focus;
```

Comment:
The blur radius looks a little funny here, could it make sense to set that to zero so the shadow more closely reflects the shape of the options list? Or are rounded box-shadows best practice in this situation?

Reply:
I copied this from the existing autosuggest styles. I think we can make another issue to ask if we should tweak them and share the css between the two places where it's used.

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5977